src/API/TokenExchangeV3ToV5.php

```diff
@@ -102,11 +102,6 @@ public static function token_update() {
 			'advertiser_id' => Pinterest_For_Woocommerce()::get_setting( 'tracking_advertiser' ),
 			'tag_id'        => Pinterest_For_WooCommerce()::get_setting( 'tracking_tag' ),
 			'merchant_id'   => Pinterest_For_Woocommerce()::get_data( 'merchant_id' ),
-			'feature_flags' => array(
```

Comment:
Just a question. We use the featgure_flats to set settings via `apply_oauth_flow_features` method. Are those settings preserved correctly from the V3 version in case of migration?